assets/js/components/Dashboard/NewPaMessage/NewPaMessagePage.tsx

```diff
@@ -314,4 +389,92 @@ const ReviewAudioButton = ({
   );
 };
 
+interface NewPaMessageHeaderProps {
+  associatedAlert: AlertModel;
+  visualText: string;
+  phoneticText: string;
+  navigateTo: (page: Page) => void;
+  setImportMessage: Dispatch<SetStateAction<boolean>>;
+  setImportLocations: Dispatch<SetStateAction<boolean>>;
+  setEndWithEffectPeriod: Dispatch<SetStateAction<boolean>>;
+  setAssociatedAlert: Dispatch<SetStateAction<AlertModel>>;
+  setVisualText: Dispatch<SetStateAction<string>>;
+  setPhoneticText: Dispatch<SetStateAction<string>>;
+  setEndDate: Dispatch<SetStateAction<string>>;
+  setEndTime: Dispatch<SetStateAction<string>>;
+}
+
+const NewPaMessageHeader = ({
+  associatedAlert,
+  navigateTo,
+  setImportMessage,
+  setImportLocations,
+  setEndWithEffectPeriod,
+  setAssociatedAlert,
+  setVisualText,
+  setPhoneticText,
+  setEndDate,
+  setEndTime,
+}: NewPaMessageHeaderProps) => {
+  const now = moment();
+  const formatActivePeriod = (activePeriods: ActivePeriod[]) => {
+    const [start, end] = getAlertEarliestStartLatestEnd(activePeriods);
+    return (
+      <div className="affect-period">
+        Alert Effect period: {start} - {end}
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <Row md="auto" className="align-items-center">
+        {associatedAlert.id ? (
```

Comment:
suggestion: It might be a bit easier to follow if you wrap the whole component in this conditional. So basically:

```ts
return {associatedAlert.id ? AssociatedAlertStuff : NotAssociatedAlertStuff}
```